dynamodb-stream-reader/src/test/java/no/unit/nva/dynamodb/IndexFilterBuilderTest.java

```diff
@@ -0,0 +1,33 @@
+package no.unit.nva.dynamodb;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.function.Predicate;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class IndexFilterBuilderTest {
+
+
+    public static final String INDEX = "tulliball";
+
+    @Test
+    @DisplayName("Testing building of Indexfiler wuth indexed")
+    public void testIndexFilterWithIndex() {
```

Comment:
test name should describe the tested functionality.
More about testing here: https://github.com/lemon123456/Books/blob/master/Pragmatic%20Unit%20Testing%20in%20Java%208%20with%20JUnit.pdf